### ISSUE TITLE
Add feature to send large files.

### ISSUE THREAD

**Issue by firtysh** [firtysh/real-time-file-share#7]
Currently its not possible to send larger files through the socket connection. Add a feature so that user can send large files.

**Comment by mecivil** [comment]
I want to contribute to this under JWOC

**Comment by firtysh** [comment]
> I want to contribute to this under JWOC

Sorry i can not assign the issue untill your previous pr get merged

**Comment by RishabhJain0721** [comment]
I want to contribute to this under JWOC

**Comment by firtysh** [comment]
> I want to contribute to this under JWOC

assigned

**Comment by firtysh** [comment]
> I want to contribute to this under JWOC

Any progress @RishabhJain0721

**Comment by RishabhJain0721** [comment]
@firtysh I will be raising a PR by tomorrow positively

**Comment by RishabhJain0721** [comment]
@firtysh Sorry I think I will not be able to do it today.You may assign it to the next person if you wish.

**Comment by firtysh** [comment]
> @firtysh Sorry I think I will not be able to do it today.You may assign it to the next person if you wish.

No problem

**Comment by mecivil** [comment]
I want to contribute under JWOC

**Comment by firtysh** [comment]
> I want to contribute under JWOC

Assigned

**Comment by deepbeatz** [comment]
I want to work on this issue under JWOC

**Comment by LuckySingh9897** [comment]
I want to work on this issue under JWOC.

**Comment by mecivil** [comment]
I will be making PR today. I have got it done.Please don't unassign me.i know it's a little late.Sorry

**Comment by firtysh** [comment]
> I will be making PR today. I have got it done.Please don't unassign me.i know it's a little late.Sorry

Yeah i checked your pr it was a good approach but you totally modified the code. Sorry but i had to close the pr.
As per the rules i have to assign the issue to the next person but no worries you can try again if they also fail to solve the issue.

**Comment by firtysh** [comment]
> I want to work on this issue under JWOC

Assigned

**Comment by deepbeatz** [comment]
how do i go about on this issue , can i get some guidance @firtysh

**Comment by firtysh** [comment]
> how do i go about on this issue , can i get some guidance @firtysh

send the file in chunks instead of sending the whole file data at once

**Comment by Beingfarazkhan** [comment]
I would like to give this issue a try, you can assign it to me

**Comment by ManasaKaza** [comment]
I want to work on this issue under JWOC

**Comment by firtysh** [comment]
> I would like to give this issue a try, you can assign it to me

Assigned
